src/util.rs

```diff
@@ -127,10 +127,17 @@ fn main() -> anyhow::Result<()> {
     // Attempt to connect to ptouch device to inform configuration
     let connect = match PTouch::new(&opts.options) {
         Ok(mut pt) => {
-            debug!("Connected! fetching status...");
+            let status;
+            if opts.options.no_status_fetch {
+                debug!("Connected! status request disabled, using default status...");
```

Comment:
might be nice to have this as an `info!` message because it's useful to know (and _could_ include a print of what those defaults were?)

Reply:
That's right. I changed that, so now the program will print:
```
[INFO] Connected! status request disabled, using default status...
[INFO] Device status (default one used): Status { ... }
```
or
```
[INFO] Connected! fetching status...
[INFO] Device status (fetched from device): Status { ... }
```
depending on if the `--no-status-fetch` switch is activated or not.

Tell me if you would like to change something !